src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane.PlaneBuilder plane = new Plane.PlaneBuilder().setModel("Boeing737");
+        plane = plane.setColor("white");
+        plane = plane.setLength(30);
+        plane = plane.setWide(28);
+        plane = plane.setPrice(90);
+        plane.build();
```

Comment:
```suggestion
        Plane plane = new Plane.PlaneBuilder()
                .setModel("Boeing737");
                .setColor("white");
                .setLength(30);
                .setWide(28);
                .setPrice(90);
                .build();
```
What do setters in builder return? answer in your chat(mention me please)